src/wirecloud/platform/static/js/wirecloud/ui/WiringEditor/ComponentGroup.js

```diff
@@ -129,4 +131,21 @@
         this.imageElement.parentElement.appendChild(document.createTextNode(utils.gettext("No image available")));
     };
 
+    var orderVersions = function orderVersions(versions) {
+        versions = versions.map(function (value) {
+            return new Wirecloud.Version(value);
+        }).sort(function (version1, version2) {
+            return version1.compareTo(version2);
+        }).map(function (version) {
+            return {
+                label: 'v' + version.text,
+                value: version.text
+            };
+        });
+
+        versions[versions.length - 1].label += ' ' + utils.gettext("(latest)");
```

Comment:
Please, provide a full format message suitable for translation and use `utils.interpolate`.